### DISCUSSION TITLE
Hp Sleekbook15 Loading Brunch Framework... Boot cycle

### DISCUSSION THREAD

**Discussion by chaskins81** [sebanc/brunch#817] [Q&A]
I have an HP Sleekbook15 (b107cl)with an AMD A8-4555m APU with Radeon HD Graphics, Processor Speed is 1600MHz and 8gb of ddr 3 ram. through a flashdrive loaded with mint 20.1, the latest stable build of brunch, and chromos grunt, I was able to load Chrome OS "successfully". Upon powering down, pulling the flashdrive when prompted and rebooting, I get the run Chrome OS screen hit enter, moves to Loading Brunch Framework, I then get a prompt saying " ChromeOS rootfs is being rebuilt. This is due to an update or a change in options. Please be patient, this may take a few minutes." after a few minutes have past the prompt changes to  "ChromeOS rootfs patches are being applied. Please be patient, this may take a few minutes." after a few seconds the screen flashed black and the grub menu asking if i want to load ChromeOs ChromeOS (debug mode) flashes back on. If I hit enter or just let it self load after a few seconds the Loading Brunch Framework... screen pops back up sits for about a minute system reboot back to grub, lather rinse repeat...I followed the Tutorial to a T, and I'm still pretty new to Linux but based on my personal research into this ChromeOS grunt is flashed from a HP Chromebook using the AMD A6 CPU so I don't see why an A8 cpu wouldnt work., so do a need to do something with the linux command line or did i miss something?
![20210119_104112](https://user-images.githubusercontent.com/77684342/105062538-b3770900-5a48-11eb-867a-80f8b01efa81.jpg)
![20210119_104521](https://user-images.githubusercontent.com/77684342/105062554-b7a32680-5a48-11eb-8885-8c90764274a3.jpg)
![20210119_105933](https://user-images.githubusercontent.com/77684342/105062608-cb4e8d00-5a48-11eb-99b6-997021225666.jpg)

**Comment by Warrentheo** [comment]
Re-check the compatibility section...  As a hint, you have an AMD based off of Piledriver...

**Reply by chaskins81** [reply]
Yeah, kinda gave up, I ended up doubling the ram, and swapping out the hdd with a ssd and a fresh install of 10 and gave it to my daughter...would have made a perfect chromebook with the touch screen though...which still was ment for my daughter...

**Reply by Warrentheo** [reply]
👍
Those bulldozer/piledriver APU's were just weird chips from the get-go...  I was so glad when AMD came out with Ryzen 😁
